proposals/simd/W3CTAG-SIMDExplainer.md

```diff
@@ -0,0 +1,70 @@
+# WASM SIMD Web Platform explainer
+
+## User benefits
+Modern processors support running multiple common computations in a single operation, so that instead of multiplying a number with a number, you can add (or multiple, subtract etc) the individual components of a fixed sized vector instead.
```

Comment:
```suggestion
Modern processors support running multiple common computations in a single operation, so that instead of multiplying a number with a number, you can add (or multiply, subtract, etc.) the individual components of a fixed sized vector instead.
```

Reply:
Fixed, thanks.